command/operator_members.go

```diff
@@ -70,9 +71,33 @@ func (c *OperatorMembersCommand) Run(args []string) int {
 
 	switch Format(c.UI) {
 	case "table":
-		out := []string{"Host Name | API Address | Cluster Address | ActiveNode | Last Echo"}
+		showUpgradeVersion := false
+		out := make([]string, 0)
+		cols := []string{"Host Name", "API Address", "Cluster Address", "Active Node", "Version"}
+
+		// If any of the nodes have an UpgradeVersion, show the header
+		if len(resp.Nodes) > 0 {
+			for _, node := range resp.Nodes {
+				if node.UpgradeVersion != "" && node.UpgradeVersion != node.Version {
```

Comment:
Why only if != node.Version?

Reply:
It's possible this was from an early version where `UpgradeVersion` was always set, but the gist is I didn't want to confuse users by showing redundant information, especially if they didn't explicitly set `UpgradeVersion`.

Reply:
I think it might be more confusing to have this column appear only sometimes.  My preference would be to make its presence unconditional, but I don't feel very strongly about it.